cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -529,6 +529,14 @@
     </description>
   </property>
 
+  <property>
+    <name>app.program.runtime.monitor.active.monitoring</name>
```

Comment:
This properties is only temporary until we switch to use the new RuntimeClient, which will be in in a day.